src/components/ListOfPayment.tsx

```
@@ -74,32 +112,110 @@
-            Confirmation
+        <Box sx={{ ...style, width: 400, border: 0 }}>
+          <h4 id="child-modal-title" className="font-semibold mb-5">
+            Refund <span className=" text-gray-400">- {row.report_id}</span>
           </h4>
-          <p id="child-modal-description" className="text-sm">
-            Are you sure to refund this payment?
-          </p>
-          <Button onClick={handleClose}>Confirm</Button>
+          <div className="flex flex-col overflow-y-auto thin-scrollbar mt-2 pr-2 h-60 2xl:h-96">
+            {row?.order_name.map((name: any, i: number) => (
+              <div className="mt-5 border-b-2 pb-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)